pytest 2x2 fails

Pytest with `2x2` (2 containers with 2 processes each, 4 processes in total) fails with the following error.

```
tests/chainermn_tests/communicator_tests/test_communicator.py::test_communicator_gpu[param6] FAILED

<snip>

communicator = <chainermn.communicators.two_dimensional_communicator.TwoDimensionalCommunicator object at 0x7efd4882dc50>
model = <test_communicator.ExampleModel object at 0x7efd4882dbd0>

<snip>

tests/chainermn_tests/communicator_tests/test_communicator.py:316:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

x = array([3.5, 3.5, 3.5, 5. , 5. ], dtype=float32)
y = array([3.5, 3.5, 3.5, 3.5, 3.5]), atol = 1e-05, rtol = 0.0001
verbose = True

<snip>

E           AssertionError:
E           Not equal to tolerance rtol=0.0001, atol=1e-05
E
E           Mismatch: 40%
E           Max absolute difference: 1.5
E           Max relative difference: 0.42857143
E            x: array([3.5, 3.5, 3.5, 5. , 5. ], dtype=float32)
E            y: array([3.5, 3.5, 3.5, 3.5, 3.5])
E
E           assert_allclose failed:
E             shape: (5,) (5,)
E             dtype: float32 float64
E             i: (3,)
E             x[i]: 5.0
E             y[i]: 3.5
E             relative error[i]: 0.428571428571
E             absolute error[i]: 1.5
E           x: [3.5 3.5 3.5 5.  5. ]
E           y: [3.5 3.5 3.5 3.5 3.5]

chainer/testing/array.py:59: AssertionError

```

The full output is shown here: https://gist.github.com/keisukefukuda/4e5a89b039f4e4083769ba8303bbf3b2

--- a/chainermn/communicators/two_dimensional_communicator.py
+++ b/chainermn/communicators/two_dimensional_communicator.py
@@ -54,7 +54,7 @@ class TwoDimensionalCommunicator(mpi_communicator_base.MpiCommunicatorBase):
         itemsize = 4
         n_elems_total = _memory_utility.count_grad_elements(params,
                                                             zero_fill)
-        n_elems_per_node_2d = int(math.ceil(n_elems_total / self.size))
+        n_elems_per_node_2d = int(math.ceil(n_elems_total / self.inter_size))
         n_elems_per_node_1d = n_elems_per_node_2d * self.inter_size
         n_bytes_per_node_1d = n_elems_per_node_1d * itemsize
         n_bytes_per_node_2d = n_elems_per_node_2d * itemsize

